news/2024/2024-05-10-kyoku-original-song-and-beatmapping-contest.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: "Kyoku: Original Song & Beatmapping Contest"
+date: 2024-05-10 18:00:00 +0000
+---
+
+All mappers and music producers, gather up! Creativity to the maximum using music is now possible within osu!
+
+![](/wiki/shared/news/2024-05-10-kyoku-original-song-and-beatmapping-contest/banner.jpg)
+
+Better yet, **you** can join in on the fun! A team of hosts, [bokeru](https://osu.ppy.sh/users/11308433), [chidodou](https://osu.ppy.sh/users/13868581), [Supportive](https://osu.ppy.sh/users/21482068) and [Whithard](https://osu.ppy.sh/users/24552396) have joined up to create a contest revolving around the idea of expression, versatility, and best of, **expression**! Keep reading to see how you can join and be an original participant of this contest!
+
+With the first iteration and the first of its kind in osu!, as well as being most inspired from BMS of Fighters (surely something you've heard about before), Kyoku is a contest that will focus on not only beatmapping, but song production as well. Instead of free song choice, or the usual forced song pick, you'll have to get together with a musician/musicians (or even be a producer yourself!) in teams of 2-8 (up to 5 mappers and up to 3 musicians) to create a fresh new song AND map!
+
+## Schedule
+
+| Phase | Dates |
+| --: | :-- |
+| Registrations | Now to June 9 |
+| Song production & mapping period | June 10 to August 22 |
+| Judging | August 23 to August 31 |
+| Results | September 1 |
+
+## Song rules
+
+Starting June 10th, 2024, musicians will be allowed to send their song to any of the organisers to check over for approval, under these rules:
+
+- **Songs must be original works created for the contest**, or at the very least, previously unreleased. Anything found to be breaking this rule will be immediately disqualified.
+- **Songs must stay unreleased until the day of results.**
+- **Songs must be 2:00–6:00 in drain time.**
```

Comment:
drain time only makes sense for beatmaps. this is supposed to just mean song length, right?